pkg/sfu/buffer/dependencydescriptorparser.go

```diff
@@ -28,6 +29,20 @@ import (
 	"github.com/livekit/protocol/logger"
 )
 
+var (
+	ExtDependencyDescriptorFactory *sync.Pool
+)
+
+func init() {
```

Comment:
these don't need an init function. the variable can be initialized in the declaration